estuary_updater/handlers/bugzilla.py

```diff
@@ -0,0 +1,86 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+from __future__ import unicode_literals, absolute_import
+
+from estuary.utils.general import timestamp_to_datetime
+from estuary.models.bugzilla import BugzillaBug
+from estuary.models.user import User
+
+from estuary_updater.handlers.base import BaseHandler
+
+
+class BugzillaHandler(BaseHandler):
+    """A handler for Bugzilla bug related messages."""
+
+    @staticmethod
+    def can_handle(msg):
+        """
+        Determine if this handler can handle this message.
+
+        :param dict msg: a message to be analyzed
+        :return: a bool based on if the handler can handle this kind of message
+        :rtype: bool
+        """
+        supported_topics = [
+            '/topic/VirtualTopic.eng.bugzilla.bug.modify',
+            '/topic/VirtualTopic.eng.bugzilla.bug.create',
+        ]
+        return msg['topic'] in supported_topics
+
+    def handle(self, msg):
+        """
+        Handle a Bugzilla bug message and update Neo4j if necessary.
+
+        :param dict msg: a message to be processed
+        """
+        topic = msg['topic']
+
+        bug_topics = [
+            '/topic/VirtualTopic.eng.bugzilla.bug.modify',
+            '/topic/VirtualTopic.eng.bugzilla.bug.create',
+        ]
+
+        if topic in bug_topics:
```

Comment:
Couldn't you just call `if self.can_handle(msg):`?

Reply:
Originally I had plans to make another handler for a different kind of bug message that I am not currently handling, but since it isn't there this makes more sense, yea :)